spacy/tokens/doc.pyx

```diff
@@ -884,7 +884,7 @@ cdef class Doc:
 
         if attrs is None:
             attrs = [LEMMA, NORM]
-            if all(doc.is_nered for doc in docs):
+            if any(doc.is_nered for doc in docs):
                 attrs.extend([ENT_IOB, ENT_KB_ID, ENT_TYPE])
```

Comment:
Agreed that `any` makes more sense! What about the cases for `is_tagged` and `is_parsed` just a few lines below this one - shouldn't they be fixed in a similar fashion too?